trueos-handbook/introducing.rst

```diff
@@ -1019,9 +1130,10 @@ These can be used to retrieve earlier versions of files, rollback the
 operating system to an earlier point in time, or clone the operating
 system.
 
-More RAM is always recommended, so install as much as you can afford.
-To play modern video games, use a fast CPU. To create a collection of
-music and movies on the computer, sufficient disk space is required.
+There is no such thing as to much RAM. ZFS thrives on systems with lots
```

Comment:
"to" -> "too"

Reply:
Ouch, this one is embarrassing... At least it wasn't "two much RAM" :)